community/eigen/APKBUILD

```diff
@@ -1,36 +1,49 @@
 # Contributor: Bradley J Chambers <brad.chambers@gmail.com>
 # Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+# Contributor: stef <l0ls0fo2i@ctrlc.hu>
```

Comment:
Add yourself to the top, not the bottom of the contributor graph.

Reply:
good to know. fixed.